jr-record-test/src/test-jdk17/java/jr/Java17RecordTest.java

```diff
@@ -1,22 +1,137 @@
 package jr;
 
+import java.io.IOException;
 import java.util.Map;
 
 import com.fasterxml.jackson.jr.ob.JSON;
 
+import com.fasterxml.jackson.jr.ob.JSON.Feature;
 import junit.framework.TestCase;
 
 /**
  * This test is in test module since the JDK version to be tested is higher than other, and hence supports Records.
  */
 public class Java17RecordTest extends TestCase
 {
+
+    private final JSON jsonParser = JSON.builder().enable(Feature.USE_FIELD_MATCHING_GETTERS).build();
```

Comment:
No, as per earlier comments should not require enabling of this setting.

Will change setting back.